code/src/Components/UserInput.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from 'react'
+
+import { useDispatch } from 'react-redux'
+
+import { tasks } from '../Reducers/tasks'
+
+import { Button } from '../library/Button'
+import { UserInputBox, InputText } from '../library/Containers'
+
+export const UserInput = () => {
+  const [task, setTask] = useState('')
+  const [addBtn] = useState(true)
+
+  const dispatch = useDispatch()
+
+  const handleSubmit = (event) => {
+    event.preventDefault()
+    dispatch(
+      tasks.actions.addItem({
+        taskInfo: {
+          id: Date.now(),
```

Comment:
A more used way of handling id is to use uuid library. You will then have to do a npm install and import that library to this component.  
Installation: `npm install uuid` I
mport: `import { v4 as uuidv4 } from 'uuid';` 
Call function:  `uuidv4() `  instead of Date.now() on line 21.